TYPO3.Neos/Classes/TYPO3/Neos/Command/DomainCommandController.php

```diff
@@ -94,115 +94,88 @@ public function addCommand($siteNodeName, $hostPattern, $scheme = null, $port =
      *
-     * @param string $hostPattern An optional host pattern to search for
+     * @param string $hostname An optional host name to search for
      * @return void
      */
-    public function listCommand($hostPattern = null)
+    public function listCommand($hostname = null)
     {
-        if ($hostPattern === null) {
+        if ($hostname === null) {
             $domains = $this->domainRepository->findAll();
         } else {
-            $domains = $this->domainRepository->findByHost($hostPattern);
+            $domains = $this->domainRepository->findByHostname($hostname);
         }
 
         if (count($domains) === 0) {
-            $this->outputLine('No domains available.');
+            $this->outputLine('No domain entries available.');
             $this->quit(0);
         }
 
-        $longestNodeName = 9;
-        $longestHostPattern = 12;
-        $availableDomains = array();
-
+        $availableDomains = [];
         foreach ($domains as $domain) {
             /** @var \TYPO3\Neos\Domain\Model\Domain $domain */
-            array_push($availableDomains, array(
+            $availableDomains[] = [
                 'nodeName' => $domain->getSite()->getNodeName(),
-                'hostPattern' => $domain->getHostPattern(),
-                'scheme' => $domain->getScheme(),
-                'port' => $domain->getPort(),
-                'active' => $domain->getActive()
-            ));
-            if (strlen($domain->getSite()->getNodeName()) > $longestNodeName) {
-                $longestNodeName = strlen($domain->getSite()->getNodeName());
-            }
-            if (strlen($domain) > $longestHostPattern) {
-                $longestHostPattern = strlen($domain);
-            }
+                'hostname' => (string)$domain,
+                'active' => $domain->getActive() ? 'active' : 'inactive'
+            ];
         }
 
-        $this->outputLine();
-        $this->outputLine(' ' . str_pad('Node name', $longestNodeName + 10) . str_pad('Domain (Scheme/<b>Host</b>/Port)', $longestHostPattern + 12) . 'State');
-        $this->outputLine(str_repeat('-', $longestNodeName + $longestHostPattern + 10 + 2 + 12));
-        foreach ($availableDomains as $domain) {
-            $this->outputLine(sprintf(
-                ' %s%s%s',
-                str_pad($domain['nodeName'], $longestNodeName + 10),
-                str_pad(
-                    ($domain['scheme'] ? $domain['scheme'] . '://' : '') .
-                    '<b>' . $domain['hostPattern'] . '</b>' .
-                    ($domain['port'] ? ':' . $domain['port'] : ''),
-                    $longestHostPattern + 12
-                ),
-                ($domain['active'] ? 'Active' : 'Inactive')
-            ));
-        }
-        $this->outputLine();
+        $this->output->outputTable($availableDomains, ['Node name', 'Domain (Scheme/Host/Port)', 'State']);
     }
 
     /**
-     * Delete a domain record
+     * Delete a host (domain) record
```

Comment:
this doesn't make sense to me.. it's deleting a domain not a host.. could be `Delete a domain record by hostname` if it needs to be more clear